resources/stsci.edu/schemas/general_sersic2d-1.0.0.yaml

```diff
@@ -0,0 +1,64 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/general_sersic2d-1.0.0"
+title: >
+  Two dimensional generalized Sersic surface brightness profile.
+
+description: >
+  Two dimensional Sersic surface brightness profile over a generalized ellipse.
+
+examples:
+  -
+    - $I(x, y)=I(r)=I(x, y)=I(r)=10.0\exp\left\{-b_n\left[\left(\frac{(\sqrt{(x-0.5)^2 + (y-1.5)^2})}{1.0}\right)^{(1/4)}-1\right]\right\}$ where $b_n$ is defined such that $r_e$ contains half the total luminosity (can be solved for numerically).
+    - |
+      !transform/sersic2d-1.0.0 {amplitude: 10.0, ellip: 0.0, n: 4.0, r_eff: 1.0, theta: 0.0, x_0: 0.5, y_0: 1.5}
```

Comment:
Would you update these to use `general_sersic2d-1.0.0` instead of `sersic2d-1.0.0`?